### ISSUE TITLE
Is this possible to run

### ISSUE THREAD

**Issue by Bazimek** [Blyzz616/PZ-Disco#1]
If server is running on Pterodactyl panel?

**Comment by Blyzz616** [comment]
> If server is running on Pterodactyl panel?

No. The code is  all in Bourne Again Shell (BASH) .  This means that it runs on the operating system level - specifically on a Linux distro. I believe that Pterodactyl panel runs in a browser engine? (I may be mistaken).
